.travis.yml

```diff
@@ -7,11 +7,11 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_CMS=1.0.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
-      env: DB=MYSQL PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_CMS=1.1.x-dev PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL RECIPE_CMS=1.x-dev PHPUNIT_COVERAGE_TEST=1
```

Comment:
My only thought about this (relevant: #725) is that we're likely to use this as a relatively abstract example of matrix testing and copy it to other modules, so calling it RECIPE_CMS isn't necessarily always true, whereas RECIPE_VERSION in #725 is broader - it still doesn't cover the installer though. Do far all of the other env vars here are the same across all modules though

Reply:
I used the name as it is specific to the module we're after.
e.g. In the changes I'm WIPing for `comments` I've used `$INSTALLER` in relation to the requirement there: https://github.com/silverstripe/silverstripe-comments/compare/master...creative-commoners:pulls/3.0/enhance-integration

I've no qualms if you'd prefer me to change it/us to standardise on a particular name.

Reply:
I think RECIPE or RECIPE_VERSION would be good so we can re-use it. We can use something different in cases where it uses the installer instead. I'd like to try and keep Travis configs as uniform as possible

Reply:
sweet